LocationServices/LocationServices/Views/Common/CommonDashboardView.swift

```diff
@@ -95,17 +96,19 @@ final class CommonDashboardView: UIView {
                                     buttonTitle: String,
                                     titleFont: UIFont?,
                                     detailLabelFont: UIFont?) {
+        self.backgroundColor = .searchBarBackgroundColor
         self.titleLabel.text = title
         self.titleLabel.font = titleFont
         self.detailLabel.text = detail
         self.detailLabel.font = UIFont.amazonFont(type: .regular, size: 13)
         self.iconView.image = image
         self.iconView.backgroundColor = iconBackgroundColor
+        self.iconView.tintColor = .black
         self.comonButton.setTitle(buttonTitle, for: .normal)
     }
     
     func hideMaybeLaterButton(state: Bool) {
-        self.maybeLaterButton.isHidden = state
+        self.maybeLaterButton.isHidden = true // state
```

Comment:
debug code?

Reply:
No it is asked not to show maybe later button. I've not removed it but just hid it

Reply:
No it is asked not to show maybe later button. I've not removed it but just hid it